site/docs/configuration.md

```diff
@@ -91,7 +91,7 @@ Iceberg tables support table properties to configure table behavior, like the de
 | commit.retry.num-retries           | 4                | Number of times to retry a commit before failing              |
 | commit.retry.min-wait-ms           | 100              | Minimum time in milliseconds to wait before retrying a commit |
 | commit.retry.max-wait-ms           | 60000 (1 min)    | Maximum time in milliseconds to wait before retrying a commit |
-| commit.retry.total-timeout-ms      | 1800000 (30 min) | Maximum time in milliseconds to wait before retrying a commit |
+| commit.retry.total-timeout-ms      | 1800000 (30 min) | Total timeout in milliseconds between retrying a commit.                              |
```

Comment:
```suggestion
| commit.retry.total-timeout-ms      | 1800000 (30 min) | Total timeout in milliseconds when retrying a commit.                              |
```
I think this is a better description of what this does?

Reply:
Think about it, it makes sense.I will modify this description. Thanks.